config/config-sample.js

```diff
@@ -1,5 +1,7 @@
 module.exports = {
-    base_url: 'http://127.0.0.1:3000',
+    port: 36902, // for production environment
+    dev_port: 3000, // for gulp watch
+    base_url: 'http://127.0.0.1:3000/',
```

Comment:
The base_url is used when coming back from Paytrail. It could be renamed to something like 'public_url' instead. When devving locally, you need to have the dev port there unless you're using reverse proxy or express is on port 80.

The thing about two ports is true, though - one would be sufficient.

IMO we don't need config for dev/prod, as dev should be running through gulp and prod should run independently. 

-----Original Message-----
From: "Pekko Lipsanen" <notifications@github.com>
Sent: ‎12/‎10/‎2015 12:44
To: "teekkarispeksi/nappikauppa2" <nappikauppa2@noreply.github.com>
Cc: "Arttu Voutilainen" <arttu.voutilainen@gmail.com>
Subject: Re: [nappikauppa2] Changes required to make reverse proxy work (#27)

In config/config-sample.js:
> @@ -1,5 +1,7 @@
>  module.exports = {
> -    base_url: 'http://127.0.0.1:3000',
> +    port: 36902, // for production environment
> +    dev_port: 3000, // for gulp watch
> +    base_url: 'http://127.0.0.1:3000/',
Something feels wrong here even if it's only an example - why does base_url has dev_port in it? Why two ports? How about: {port: .. , base_url: .. , mode: "dev" or "prod"}, where autoreload/watch is enabled if mode==dev (maybe logging settings could also change)
—
Reply to this email directly or view it on GitHub.